### ISSUE TITLE
UI/UX: Select module dialog when opening foreign file is confusing

### ISSUE THREAD

**Issue by leoheck** [FreeCAD/FreeCAD#16431]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Problem description

First of all, the title of the pop up window is a bit off.
Suggestion: `SVG Import Settings`

![image](https://github.com/user-attachments/assets/9f0c5c87-1888-40e4-a399-420aaad447aa)

Second, it seems that the operations are switched.

- Selecting the first one `Image formats (Freecad)`, imports an image, but it is written (Freecad).
- The second option `Import geometry (importSVG)`, create some Part Objects.

The text if those options are misleading and could be improved. I have no suggestion for this since I am not sure if the operation of each item is right. 

Extra: Tool-tips for each option would be a great addition too.

### Full version info

```shell
OS: Ubuntu 24.04.1 LTS (ubuntu:GNOME/ubuntu-xorg)
Word size of FreeCAD: 64-bit
Version: 1.0.0RC1.38643 (Git) AppImage
Build type: Release
Branch: main
Hash: fe40748c10503eed367840239db3e0f81873e359
Python 3.11.9, Qt 5.15.13, Coin 4.0.2, Vtk 9.2.6, OCC 7.7.2
Locale: English/United States (en_US)
```

### Subproject(s) affected?

Core

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by Roy-043** [comment]
The dialog informs that there are 2 modules (FreeCAD and importSVG) that can import an SVG file. The same dialog is used whenever more than one import module is available for a file format.

**Comment by leoheck** [comment]
What are those modules? Code? I don't think those names make the end user to understand the operation of each item better.

**Comment by luzpaz** [comment]
How do we give the end-user here a context to each choice ? Should we have links to the docs here ?

**Comment by leoheck** [comment]
I think the options should be rewritten to be clear, removing those module names. It the module names are important, then adding something as a tool-tip that explains that would be enough.

I would suggest something....

Open SVG as...
- `an Image`
- `a Part object`, `a Freecad object`, `a Freecad geometry`

**Comment by luzpaz** [comment]
@Roy-043 what say you ?

**Comment by Roy-043** [comment]
Core is displaying the information it has available. Which basically matches what you see in the Std_Import dialog. For me this is enough. It is also consistent I would say.

Module names should stay in the dialog IMO. You might have two modules that can import DXF files, where their results are similar, but in some cases one works better than the other.

**Comment by ElementW** [comment]
@leoheck This was autoclosed but the situation is unchanged as of `1.1.0dev.42885` (2025/08/08 15:11:51), could you reopen?

@Roy-043 If this is the result of Core displaying what information it has, then it means that information is insufficient for a user (who doesn't know FC's inner machinations) to make an informed decision. Keeping the module names in that window is not incompatible with having more descriptive labels. One could even extend this with icons to illustrate what is going to happen, for example:

<img width="341" height="200" alt="Image" src="https://github.com/user-attachments/assets/b94fedbb-5c9f-494f-9596-204451fda549" />
<img width="341" height="200" alt="Image" src="https://github.com/user-attachments/assets/b4580719-da9e-4cf5-910a-3105fb8b0d02" />

**Comment by leoheck** [comment]
Yeah, these things are getting closed automatically. But I cannot reopen them. 

Your suggestion is interesting.

I would just add that the image when open the svg as image should have a rectangular/square background so it will appear more or less as a picture instead of just a red circle and the blue square.

**Comment by ElementW** [comment]
Yeah I've been pondering how to accurately represent an *image object* there; then again these are just example icons. Sucks you don't have the rights to reopen an issue you created in the first place; who can then?

<img width="341" height="200" alt="Image" src="https://github.com/user-attachments/assets/35228e82-b89e-4879-88d1-95a5f96433d9" />

**Comment by leoheck** [comment]
@luzpaz could you reopen this issue for us?

**Comment by leoheck** [comment]
@ElementW can you use a square with round corners. and a darker image bg?

**Comment by ElementW** [comment]
@leoheck Like this?

<img width="341" height="200" alt="Image" src="https://github.com/user-attachments/assets/b45915b5-d3e8-44a1-845b-121a79ff7b10" />

![Image](https://github.com/user-attachments/assets/bfe516dd-ca4c-4463-b3d3-b097a9f519fa)

**Comment by ElementW** [comment]
I only focused on adding example icons but hadn't given more thought the whole dialog window itself.

This would already be a lot better as you're immediately presented with a representation of each option, there's more space to add text if needs be (or even a config sub-button), it's much more forgiving to cursor position/better on tablet input, is easier to translate as strings aren't broken up/context sensitive, and saves a click.

Of course the icon to text size ratio is not ideal, but this is a mockup.

<img width="280" height="244" alt="Image" src="https://github.com/user-attachments/assets/3830e9af-cb27-467e-afc3-1d1654a58a20" />

**Comment by leoheck** [comment]
Why does it have to show `(FreeCAD plugin)` and `(importSVG plugin)`? It seems useless for the end users. The suggestion itself, with images, helps to decide it quickly. Alternatively, it could have a description or a tooltip, but this text in parentheses doesn’t help at all.

**Comment by ElementW** [comment]
🤷‍♀️ It's indeed not fundamentally useful (except for development I guess), as even if one was to write/use a plugin that provides duplicate functionality (say, importing SVG geometry through an Inkscape filter), the import string should be made different.

As it stands the import strings are extracted from the file type filters used in actual file open/import dialogs, and this is obviously lacks flexibility, plus the majority of them are untranslated and unable to be right now: #23124.

**Comment by ElementW** [comment]
Attempt to fix the design issue I outline in #23123 at the same time I guess

<img width="280" height="178" alt="Image" src="https://github.com/user-attachments/assets/539dcebf-6554-4167-b710-004bafa2a15c" />

<img width="280" height="178" alt="Image" src="https://github.com/user-attachments/assets/da57202d-8200-4aed-b274-04635fdf003c" />
